.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "qgis.installationRoot": "C:\\Program Files\\QGIS 3.34.10",
```

Comment:
When using OSGEO4W installer which is recommended by TEKSI, the path will be different.
eg : C:\\OSGeo4W\\bin\\qgis-ltr-bin.exe

Reply:
Q: Should this recommendation be placed within the README.md as well?
The settings.json was intended as a template. User can overwrite the [settings in vscode](https://code.visualstudio.com/docs/getstarted/settings). See also comment in README.md:

> 3. Install QGIS in Version 3.34.10 or adapt `qgis.installationRoot` in ['.vscode/settings.json'.](.vscode/settings.json)

If QGIS & Shell would be found on PATH on all Plattform, the configuration would be much simpler. Alternatively, if the required version / distribution of QGIS could be specified we would have the option to write a [platform aware launch.json](https://code.visualstudio.com/docs/editor/debugging#_platformspecific-properties) without using the "hacky" settings.json.

Reply:
If we adapt .vscode/settings.json will it not always show as modified in git status? 

> If QGIS & Shell would be found on PATH on all Plattform, the configuration would be much simpler.

Maybe the needed path could be defined in system variables. If not set, just use the most common variant (porbably windows/OSGeo4w ?). 
Or providing the file as a `settings.json.default` to be renamed?

But I am not used to this kind of vscode configuration I don't now what's the best way.

Reply:
I propose to use OSGeo4w as default in settings.json and adapting it to `qgis.installationRoot.windows`. If we run int in vscode the value either is correct, or the developer will get a message like "C:\OSGeo4w\qgis.bar was not found." I assume every developer will get a hint of what to do. OSGeo4w, in the optimal case, defines a link for the lates installed QGIS version to use. Corresponding linux variant can then be added as well.

The approuch should not depend on any environment variables defined from outside of VSCode as this break the whole concept of "no setup needed launch configuration".